src/vl-select.js

```diff
@@ -61,6 +61,24 @@ export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelect
     }
   }
 
+  _dataRequiredChangedCallback(oldValue, newValue) {
+    if (newValue && !oldValue) {
+      this.setAttribute('required', '');
+    }
+    if (!newValue && oldValue) {
+      this.removeAttribute('required');
+    }
+  }
+
+  _requiredChangedCallback(oldValue, newValue) {
+    if (newValue && !oldValue) {
+      this.setAttribute('data-required', '');
+    }
+    if (!newValue && oldValue) {
+      this.removeAttribute('data-required');
+    }
+  }
```

Comment:
Deze logica zorgt ervoor dat de `checkValidity` initieel ook correct is, maar een oplossing voor dit probleem kan beter op vl-form-validation niveau voorzien worden gezien het een globaal probleem is. Dit heeft op zich ook niets te maken met het `disabled = true` probleem waarvoor deze PR initieel aangemaakt werd.

Reply:
Ik onderzoek op vl-form-validation niveau of hier een globale oplossing voor toegepast kan worden, want door gebruik van validate.js is de `checkValidity` by default steeds true: https://validatejs.org/examples.html.

Reply:
@pascalriquier de logica werd verplaatst naar vl-form-validation. De unit test die je voorzien hebt is nog groen na het verwijderen van de required logica op vl-select niveau en upgrade naar de nieuwste vl-form-validation versie.